tests/unit/findUnusedStylesTest.ts

```diff
@@ -0,0 +1,385 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access */
```

Comment:
I don't feel strongly, I'd rather not disable these particular lint rules. Can't you just do `as CorrectType` assertions as needed?